errands-routes.go

```diff
@@ -251,3 +252,50 @@ func (s *ErrandsServer) GetErrandsBy(fn func(*schemas.Errand) bool) []schemas.Er
 
 	return errands
 }
+
+func (s *ErrandsServer) clearErrands(c *gin.Context) {
+	var procErrand schemas.Errand
+
+	duration, err := time.ParseDuration(c.Param("duration"))
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"message": "Invalid Duration",
+			"error":   err.Error(),
+		})
+
+		return
+	}
+
+	threshold := time.Now().Add(0 - duration)
```

Comment:
fyi 
```suggestion
	threshold := time.Now().Add(-duration)
```

works as well